cpmpy/solvers/gurobi.py

```diff
@@ -478,6 +483,14 @@ def solveAll(self, display=None, time_limit=None, solution_limit=None, call_from
             Returns: number of solutions found
         """
 
+        # ensure all vars are known to solver
+        self.solver_vars(list(self.user_vars))
+
+        # edge case, empty model, ensure the solver has something to solve
+        if not len(self.user_vars):
+            iv = intvar(1,1)
+            self += iv == iv
```

Comment:
why not `== 1` here too? as in the ortools case?

Reply:
Seems to also work.